### PR TITLE
Small typo fix + Consistent Docstrings

### PR DESCRIPTION
**Pull request by kayjay-is-here**
Also removed a redundant comment.

### CODE REVIEW

**Comment by kayjay-is-here** [comment]
There's really no point to this, but I mean it doesn't really hurt to make it look nicer.

File: backend/oasst_backend/config.py [hunk 1]
```diff
@@ -77,7 +77,7 @@ class Settings(BaseSettings):
     DEBUG_USE_SEED_DATA_PATH: Optional[FilePath] = (
         Path(__file__).parent.parent / "test_data/realistic/realistic_seed_data.json"
     )
-    DEBUG_ALLOW_SELF_LABELING: bool = False  # allow users to label their own messages
```
Comment (andreaskoepf) [inline backend/oasst_backend/config.py:80]: Thanks for the replies typo fix, but do you think self-labeling is really self-explanatory and it is better to remove the comment?

File: backend/oasst_backend/config.py [hunk 2]
```diff
@@ -20,28 +20,28 @@ class TreeManagerConfiguration(BaseModel):
...
     goal_tree_size: int = 15
-    """Total number of messages to gather per tree"""
+    """Total number of messages to gather per tree."""
 
     num_reviews_initial_prompt: int = 3
     """Number of peer review checks to collect in INITIAL_PROMPT_REVIEW state."""
 
     num_reviews_reply: int = 3
-    """Number of peer review checks to collect per reply (other than initial_prompt)"""
+    """Number of peer review checks to collect per reply (other than initial_prompt)."""
```
Comment (andreaskoepf) [inline backend/oasst_backend/config.py:29]: (I omitted the full-stop at the end of these comments because they are not full sentences (a verb is missing))

**Review by andreaskoepf — CHANGES_REQUESTED** [review]
I'd be happy to merge the docstring changes but I don't agree that the self-labeling comment is redundant and should be removed.

**Comment by kayjay-is-here** [comment]
@andreaskoepf any idea why pre-commit suddenly fails? This doesn't make any sense

**Review by andreaskoepf — APPROVED** [review]
Thanks!

**Pull request merged**